db/migrate/20160208190731_add_subtype_to_measurement_imports.rb

```diff
@@ -0,0 +1,7 @@
+class AddSyTypeToMeasurementImports < ActiveRecord::Migration
```

Comment:
The name of this migration does not match the filename, therefore this migration will throw an error when executed.